app/views/admin/edition_workflow/confirm_unpublish.html.erb

```diff
@@ -46,10 +46,43 @@
 
     <%= hidden_field_tag 'unpublishing[unpublishing_reason_id]', UnpublishingReason::Withdrawn.id %>
     <fieldset>
+      <% if @previous_withdrawals.any? %>
+        <h4>Do you need to reuse a previous withdrawal?</h4>
+        <p>You should only reuse a previous withdrawal date and public explanation if you have:</p>
+        <ul>
+          <li>updated file attachments to mark them ‘withdrawn’</li>
+          <li>made a minor edit, for example fixing a broken link</li>
+          <li>fixed an error or mistake in the content which existed when it was originally withdrawn</li>
+        </ul>
+
+        <% @previous_withdrawals.each do |withdrawal| %>
+          <div class="radio">
+            <%= label_tag "use_previous_withdrawal_#{withdrawal.id}" do %>
+              <% checked = (@use_previous_withdrawal == withdrawal.id) %>
+              <%= radio_button_tag 'use_previous_withdrawal', withdrawal.id, checked, id: "use_previous_withdrawal_#{withdrawal.id}" %>
+              <strong><%= absolute_date(withdrawal.unpublished_at) %></strong>
+              <p>“<%= withdrawal.explanation %>”</p>
+            <% end %>
+          </div>
+        <% end %>
+
+        <div class="radio">
+          <%= label_tag "use_previous_withdrawal_new" do %>
+            <% checked = (@use_previous_withdrawal == 'new') %>
+            <%= radio_button_tag 'use_previous_withdrawal', "new", checked, id: "use_previous_withdrawal_new" %>
+            <strong>This is a new withdrawal</strong>
+          <% end %>
+        </div>
+      <% else %>
+        <%= hidden_field_tag 'use_previous_withdrawal', 'new' %>
```

Comment:
Do we actually need to pass this? I wonder if we can infer that the lack of this parameter (or it being empty) is a sign of a new withdrawal.

Reply:
The presence of the parameter, but it being empty, would indicate that the user was shown radio inputs but _didn't choose one_ – i.e. it's a validation error.

I suppose we could infer it server side though by checking for `@previous_withdrawals.any?` – in other words, just repeat the same conditional check that causes this to appear in the view.

Reply:
Sorry I should have been a bit more verbose. I wondered about losing this one and having the one that is currently 'new' above be blank.

I hadn't clocked too much about the validation error though. However could validate that it's not nil rather than blank. 

Using server side check seems better for the validation given there's not anything a user can do with the validation error if they have no withdrawals to choose from.